src/views/Region/index.tsx

```diff
@@ -149,14 +149,16 @@ export function Component() {
                 >
                     {strings.regionRiskTab}
                 </NavigationTab>
-                <NavigationTab
-                    to={generatePath(
-                        regionPreparednessRoute.absolutePath,
-                        { regionId },
-                    )}
-                >
-                    {strings.regionPreparednessTab}
-                </NavigationTab>
+                {(regionResponse && regionResponse?.preparedness_snippets.length > 0) ? (
+                    <NavigationTab
+                        to={generatePath(
+                            regionPreparednessRoute.absolutePath,
+                            { regionId },
+                        )}
+                    >
+                        {strings.regionPreparednessTab}
+                    </NavigationTab>
+                ) : null}
```

Comment:
- Don't have to optional chain `regionResponse` when it's already checked
- Also check if `preparedness_snippets` is defined before accessing the length
- Even better if a variable is defined for this (e.g. `hasPreparedSnippets`)
- ? : can be replaced by &&
```suggestion
                {(regionResponse && regionResponse.preparedness_snippets && preparedness_snippets.length > 0) && (
                    <NavigationTab
                        to={generatePath(
                            regionPreparednessRoute.absolutePath,
                            { regionId },
                        )}
                    >
                        {strings.regionPreparednessTab}
                    </NavigationTab>
                ) }
```